src/routes/(demos)/trajectory/+page.svelte

```
@@ -0,0 +1,816 @@
+      handle_trajectory_file(content, filename, 'first')
+    } catch (err) {
+      console.error(`Failed to load default trajectory:`, err)
+      // Fall back to demo trajectory if default fails
+      create_demo_trajectory()
+    }
+  })
+
+  function handle_trajectory_file(
+    content: string | ArrayBuffer,
+    filename: string,
+    target: 'first' | 'second' = 'first',
```

ESLint

[error] Strings must use backtick.

(@stylistic/quotes)